### ISSUE TITLE
add DatePicker component

### ISSUE THREAD

**Issue by gioboa** [qwikifiers/qwik-ui#123]

**Comment by gioboa** [comment]
We discussed about it in the discord channel. Here the first message of a long discussion
https://discord.com/channels/990511757091033108/1040763063533588562/1075379303455391794

**Comment by briosheje** [comment]
Should we keep the native feel of `input` or should we provide an higher level abstraction where the value of the component will be a `Date` instead of the default input value type?

**Comment by maiieul** [comment]
@iancharlesdouglas suggested basing it on flatpickr. But @diecodev has started working on a date picker already right? Will it also be a time picker @diecodev? Could you add some info in here pls 🙏

**Comment by diecodev** [comment]
@maiieul the calendar is just a date picker, the time version is something I'm gonna work after the calendar and then combine them, because I need more knowledge to be able to develop the time component. As we know, JS is not the best language to deal with dates and time.

**Comment by maiieul** [comment]
Yes that's what I was thinking. Let's focus on the date picker first 👍
